voxblox_ros/include/voxblox_ros/esdf_server.h

```diff
@@ -32,19 +32,17 @@ class EsdfServer : public TsdfServer {
   virtual void publishSlices();
   void publishTraversable();
 
-  virtual void updateMesh();
   virtual void publishPointclouds();
   virtual void newPoseCallback(const Transformation& T_G_C);
-  virtual void publishMap(const bool reset_remote_map = false);
+  virtual void publishMap(bool reset_remote_map = false);
   virtual bool saveMap(const std::string& file_path);
   virtual bool loadMap(const std::string& file_path);
 
-  /**
-   * Call updateMesh if you want everything updated; call this specifically
-   * if you don't want the mesh or visualization.
-   */
+  void updateEsdfEvent(const ros::TimerEvent& event);
+
+  /// Call this to update the ESDF based on latest state of the TSDF map.
```

Comment:
add the incremental part and that it only consideres updated parts of the TSDF map?